parlai/core/metrics.py

```diff
@@ -505,9 +493,13 @@ def compute_many(
         :return: (rouge-1, rouge-2, rouge-L)
         """
         # possible global initialization
-        global rouge
-        if rouge is None:
+        try:
+            import rouge
+        except ImportError:
+            # User doesn't have py-rouge installed, so we can't use it.
+            # We'll just turn off rouge computations
```

Comment:
Hmm should we print a warning here? (Same with the bleu imports)

Reply:
Hmm, future diff